applybn/feature_selection/bn_nmi_feature_selector.py

```diff
@@ -0,0 +1,166 @@
+from pyitlib import discrete_random_variable as drv
+from sklearn.feature_selection import SelectorMixin
+from sklearn.base import BaseEstimator
+import pandas as pd
+import numpy as np
+import logging
+import itertools
+
+
+class NMIFeatureSelectior(BaseEstimator, SelectorMixin):
```

Comment:
Typo in **Selector**
```suggestion
class NMIFeatureSelector(BaseEstimator, SelectorMixin):
```